src/main/resources/lib/enonic/asset/types/Response.d.ts

```diff
@@ -18,6 +18,7 @@ export interface Response<
 	Headers extends Record<string,string|number|(string|number)[]> = {
 		'content-type'?: string
 		'cache-control'?: string
+    'content-encoding'?: string
 		'content-security-policy'?: string
```

Comment:
in specs there is also header `vary`. I don't see it implemented

Reply:
Ah, will fix.

Reply:
Fixed